pwa/app/components/tba/allianceSelectionTable.tsx

```diff
@@ -0,0 +1,117 @@
+import { Link } from '@remix-run/react';
+import { type VariantProps, cva } from 'class-variance-authority';
+import { max } from 'lodash-es';
```

Comment:
Can't we just use `Math.max`?

Reply:
That's fine, I typically find other use cases for lodash but can remove it here